packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -324,7 +325,11 @@ const AccountTransferForm = ({
                     is_mt_transfer={is_mt_transfer}
                 />
             );
-            setSideNotes(side_notes);
+            setSideNotes([
+                <SideNote title={<Localize i18n_default_text='Notes' />} key={0}>
```

Comment:
do we need to pass title here? i see that it's already in `SideNote` component

Reply:
@farrah-deriv Removed.